### DISCUSSION TITLE
Prevent a custom item from being deleted

### DISCUSSION THREAD

**Discussion by kgish** [ueberdosis/tiptap#3029] [General Questions]
I have created an extension for tiptap and it works fine within the editor window.

However, I want it to be readonly meaning that the user cannot delete it, e.g. backspace, select and delete, etc.

How can I achieve this?

**Comment by vimtor** [comment]
@kgish did you find any solution?

**Reply by kgish** [reply]
Nopes.

**Comment by mathers101** [comment]
your use case might be different but I am using this and it works good for me. The code goes in the Extension.create settings under 'addProseMirrorPlugins': https://github.com/ueberdosis/tiptap/issues/181#issuecomment-1213455982

**Comment by its-mohammadhosein** [comment]
You can simply add `contentEditable = 'false';`  to your node, I have reverse problem, I have create a custom node which contains two editable node, now I can't handle deleting that
